.ci/chart_test.sh

```diff
@@ -19,7 +19,9 @@ source ${CHARTS_HOME}/.ci/helm.sh
 ci::create_cluster
 
 # delete the cluster
-trap 'ci::delete_cluster' 0
+# TODO: Make cluster cleanup as a sepratee steep.
```

Comment:
small typo

Reply:
@maxsxu pls update this typo then we will merge this PR.